src/services/subject.js

```
@@ -0,0 +1,38 @@
+const Subject = require('~/models/subject');
+const Category = require('~/models/category');
+
+const subjectService = {
+  getSubjects: async ({ search = '', page = 1, limit = 20 } = {}) => {
+    const query = {};
+
+    if (search) {
+      query.name = { $regex: `^${search}`, $options: 'i' };
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)